src/autolabel/labeler.py

```diff
@@ -271,7 +272,8 @@ def run(
             ):
                 # get every column except the one we want to label
                 toEmbed = chunk.copy()
-                del toEmbed[self.config.label_column()]
+                if self.config.label_column():
```

Comment:
the label column column will be present in the config but it might not be present in the chunk. We should check if self.config.label_column() is in toEmbed